simulations/NavierStokes_2D/scripts/preprocess_posteriori.jl

```diff
@@ -21,6 +21,7 @@ using CoupledNODE.NavierStokes: create_io_arrays_posteriori
 io_post = create_io_arrays_posteriori(data, setups)
 
 # Example of dimensions and how to operate with io_arrays_posteriori
+ig = 1
```

Comment:
I would avoid defining this here. 
The way in which I thought one would interact with this file is including it from `train_posteriori.jl`, where this variable is define. If we redefine it here, there will be a clash.